components/navigation/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Surbhi Kukreti" />
+    <meta name="description" content="navigation component" />
+
+    <link rel="stylesheet" href="/styleGuide/util.css" />
+    <link rel="stylesheet" href="/styleGuide/main.css" />
+
+    <link rel="stylesheet" href="./navigation.css" />
+    <link rel="stylesheet" href="/components/alert/alert.css" />
+
+    <link rel="stylesheet" href="/prism/prism.css" />
+    <script src="/prism/prism.js" defer></script>
+    <title>Navigation</title>
+  </head>
+  <body>
+    <h3>Simple Navigation bar</h3>
+
+    <div class="showcase">
+      <header class="nav-head color-bp">
+        <h2 class="logo">MyLOGO</h2>
+        <ul class="nav-links">
+          <li class="search">
+            <input type="text" placeholder="Search.." class="search-bar" />
+            <button class="search-btn">
+              <img
+                src="./icons/search.svg"
+                alt="search icon"
+                class="search-icon"
+              />
+            </button>
+          </li>
+
+          <li class="nav-link">
+            <a href=""></a
+            ><img
+              src="./icons/profile.svg"
+              alt="Profile icon"
+              class="nav-icon"
+            />Profile
+          </li>
+          <li class="nav-link">
+            <a href=""></a>
+            <img
+              src="./icons/heartFilled.svg"
+              alt="heart icon"
+              class="nav-icon"
+            />
+            Wishlist
+          </li>
+          <li class="nav-link">
+            <a href=""></a>
+            <img src="./icons/cart.svg" alt="cart icon" class="nav-icon" /> Cart
+          </li>
```

Comment:
Put elements like logo and nav-links in `<a>` tag. rest is good

Reply:
yeah thanks..done now!